### ISSUE TITLE
IDISP004 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#174]
```cs
namespace Gu.Inject.Tests
{
    using NUnit.Framework;

    public class Sandbox
    {
        [Test]
        public void METHOD()
        {
            using (var kernel = new Kernel().AutoBind<Sandbox>())
            {
                var sandbox = kernel.Get<Sandbox>();
            }
        }
    }

    public static class KernelExtensions
    {
        public static Kernel AutoBind<T>(this Kernel kernel)
        {
            return kernel;
        }
    }
}
```
